### ISSUE TITLE
.NET Core December 2020 Update - 5.0.1

### ISSUE THREAD

**Issue by rbhanda** [dotnet/core#5719]
[Release Notes](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.1/5.0.1.md)

Please report any issues you find with .NET 5.0.1, either responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [dotnet/aspnetcore](https://github.com/dotnet/aspnetcore/issues) - for ASP.NET Core questions
* [dotnet/efcore](https://github.com/dotnet/efcore/issues) - for Entity Framework Core questions
* [dotnet/sdk](https://github.com/dotnet/sdk/issues) - for CLI tools and questions
* [dotnet/runtime](https://github.com/dotnet/runtime/issues) - for runtime, API, or installer issues and questions
* [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for WinForms issues
* [dotnet/wpf](https://github.com/dotnet/wpf/issues) - for WPF issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

-----
Status of availability of SDK on Microsoft's Linux feeds: 

| Distro | 5.0.1 
|--|--|
| Ubuntu 16.04   | ✅  |
| Ubuntu 18.04| ✅  | 
| Ubuntu 20.04 | ✅  | 
| Ubuntu 20.10 | ✅  | 
| Centos 7   | ✅  | 
| Debian 9    | ✅  | 
| Debian 10   | ✅  | 
| Fedora 32   |  ✅ | 
| Fedora 33  |  ✅ | 
| OpenSUSE 15 | ✅  | 
| SLES 12     | ✅ | 
| SLES15   | ✅  | 

Note: This list refers to the Microsoft-provisioned feeds (packages.microsoft.com) and does not in any way represent direct availability in distros (eg RHEL, Fedora).

### Known Issues
If there are any issues with the December 2020 release we will track them here and check issues off as they're resolved. See the linked issues for details on progress and resolution details.

**Comment by adamralph** [comment]
FYI the [announcement](https://github.com/dotnet/announcements/issues/169) has "November" in the description.

Also, those annoucements should say "in {month}" rather than "on {month}".

E.g.

> The following .NET updates were released in December 2020.

**Comment by adamralph** [comment]
Also, there's a type in the description of _this_ issue: ".NET 5.0,1" should be ".NET 5.0.1".

**Comment by RheaAyase** [comment]
Yey, we've got checkboxes on distributions that don't even have 5.0.0 yet :D
(Please differentiate between proper support, and binary release only.)

**Comment by rbhanda** [comment]
Hi @RheaAyase The checkboxes have a note added 
`Note: This list refers to the Microsoft-provisioned feeds (packages.microsoft.com) and does not in any way represent direct availability in distros (eg RHEL, Fedora).`

**Comment by maliming** [comment]
hi @rbhanda 

I set the version of the Microsoft package globally in the `Directory.Build.props` file, but I found that not all packages have released version 5.0.1.

https://www.nuget.org/packages/Microsoft.Extensions.Logging/
https://www.nuget.org/packages/Microsoft.Extensions.Localization/

All ms packages in 3.x have the same version.  Has this rule changed in the 5.x version?

Thanks.

**Comment by rbhanda** [comment]
hi @maliming these packages were not updated with 5.0.1 Release. You can find a list of all updated nuget packages on 5.0.1 [release notes](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.1/5.0.1.md#packages-updated-in-this-release)

**Comment by martin-honnen** [comment]
The early announcements for .NET 5 features (in https://devblogs.microsoft.com/dotnet/introducing-net-5/) included a feature "Java interoperability will be available on all platforms.", this doesn't seem to have made it in any way into .NET 5. Are there any plans to have it in .NET 6? For the .NET framework and Mono IKVM provided that Java interoperability.

**Comment by FIrgolitsch** [comment]
Is there anytime line for a CentOS 8 release?

**Comment by RheaAyase** [comment]
@FIrgolitsch As a low priority bugfix 5.0.1 will be released with the January update, since it wasn't possible to build it from source until a few days ago, so there is a bit of catchup to do between 5.0.0 (released two days ago) and 5.0.1

**Comment by omajid** [comment]
> Is there any timeline for a CentOS 8 release?

Builds were already done: https://koji.mbox.centos.org/koji/packageinfo?packageID=3232. Not sure when they go live.

**Comment by RheaAyase** [comment]
You can follow RHEL/Fedora details in this tracking ticket: #5596

**Comment by phillip-haydon** [comment]
`Microsoft.AspNetCore.Authorization` 5.0.1 is broken,

Just including this in a project with no authorization causes it to fail on startup due to missing configuration.

**Comment by ferraridavide** [comment]
Can't seem to be able to create ADO.NET Entity Data Model, is it not supported?

Step to recreate: Create C# Console Application with .NET 5.0 -> Add New Item -> Data -> ADO.NET Entity Data Model -> Error: "The project's target framework does not contain Entity Framework runtime assemblies."
![image](https://user-images.githubusercontent.com/23432883/102541900-4bcc4780-40b1-11eb-9277-05663e5d4b52.png)

**Comment by BhanuKorthiwada** [comment]
Hi Team,

Previously, example with .NET Core 3.1 with every release all packages were updated like 3.1.1, 3.1.2 etc.,
We were including version like `<AspNetVerion>3.1.10</AspNetVersion>` in Directory.Build.props and use it every where.
We used to update it in one place and all packages were updated to the latest.

OrchidCore does something similar.

With .NET 5, is this changed? I see that only [few packages](https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0.1/5.0.1.md#packages-updated-in-this-release) were updated.

**Comment by shaneMLK** [comment]
I would create an Azure support ticket for this, but it costs $30 for a support membership. .NET 5 (Early Access) web app Azure deployments via git are now failing. I was able to deploy .NET 5 web apps before this release. Here's some relevant logs, I can send the full logs to anyone interested:

> remote:   D:\Program Files (x86)\dotnet\sdk\5.0.100\Sdks\Microsoft.NET.Sdk.Razor\extensions\mvc-3-0\Microsoft.AspNetCore.Mvc.Razor.Extensions.dll
> remote:   
> remote: ...............
> remote:   It was not possible to find any compatible framework version
> remote:   The framework 'Microsoft.NETCore.App', version '5.0.0' was not found.

**Comment by maxhamulyak** [comment]
@BhanuKorthiwada my team does something similar which makes it very easy to manage the dependencies. Having said that most of my projects arre still on netcoreapp3.1 since that is the latest lts version at the moment.
I came looking because for a POC I did not bother specifying target framework for it. I found that not all `Microsoft.Extensions.FileProviders` were updated, is there a reason for this change?
